lte/gateway/fabfile.py

```diff
@@ -149,7 +149,11 @@ def package(
                 'cat {}'.format(mirrored_packages_file)
                 + ' | xargs -I% sudo aptitude download -q2 %',
             )
-            run('cp *.deb ~/magma-packages')
+            try:
+                run('cp *.deb ~/magma-packages')
+            except Exception:
+                # might be a problem if no deps found, but don't handle here
+                pass
             run('sudo rm -f *.deb')
```

Comment:
Not sure we want to touch this script more than necessary, but why not just `mv` instead of `cp` and `rm`? :confused:

Reply:
Ah, sudo, possibly it would result in different file permissions...